### PR TITLE
Resolve #5835: Add keyboard shortcut for scenery remover

### LINKED ISSUE #5835: Feature Request: Add keyboard shortcut for the scenery removal tool
Pretty much everything in the top toolbar has a keyboard shortcut except the scenery removal tool.

### PR DESCRIPTION
**Pull request by Kuriibo**
Added a hotkey for the "clear scenery" window. By default, the hotkey is the "b" key (b/c bulldozer).

### CODE REVIEW

File: src/openrct2/interface/window.h [hunk 1]
```diff
@@ -497,6 +497,7 @@ enum {
 #define WC_TOP_TOOLBAR__WIDX_WATER                      9
 #define WC_TOP_TOOLBAR__WIDX_SCENERY                    10
 #define WC_TOP_TOOLBAR__WIDX_PATH                       11
+#define WC_TOP_TOOLBAR__WIDX_CLEAR_SCENERY              17
```
Comment (marijnvdwerf) [inline src/openrct2/interface/window.h]: This needs an accompanying `validate_global_widx(WC_TOP_TOOLBAR, WIDX_CLEAR_SCENERY)` in `windows/top_toolbar.c`

File: src/openrct2-ui/windows/shortcut_keys.c [hunk 2]
```diff
@@ -150,6 +150,7 @@ const rct_string_id ShortcutStringIds[] = {
     STR_SHORTCUT_RIDE_CONSTRUCTION_BUILD_CURRENT,
     STR_SHORTCUT_RIDE_CONSTRUCTION_DEMOLISH_CURRENT,
     STR_LOAD_GAME,
+    STR_CLEAR_SCENERY,
```
Comment (marijnvdwerf) [inline src/openrct2-ui/windows/shortcut_keys.c]: Needs a new shortcut-specific string

File: src/openrct2/localisation/string_ids.h [hunk 3]
```diff
@@ -1962,6 +1962,7 @@ enum {
     STR_SHORTCUT_SHOW_MAP = 2523,
     STR_SHORTCUT_SCREENSHOT = 2524,
     STR_SHORTCUT_KEY_UNKNOWN = 2525,
+
```
Comment (marijnvdwerf) [inline src/openrct2/localisation/string_ids.h]: trailing whitespace

File: src/openrct2/localisation/string_ids.h [hunk 4]
```diff
@@ -3799,6 +3800,8 @@ enum {
     STR_OBJECT_ERROR_WINDOW_EXPLANATION = 6128,
     STR_COPY_SELECTED = 6129,
     STR_COPY_ALL = 6130,
+
```
Comment (marijnvdwerf) [inline src/openrct2/localisation/string_ids.h]: trailing whitespace

File: src/openrct2/localisation/string_ids.h [hunk 5]
```diff
@@ -3799,6 +3800,8 @@ enum {
     STR_OBJECT_ERROR_WINDOW_EXPLANATION = 6128,
     STR_COPY_SELECTED = 6129,
     STR_COPY_ALL = 6130,
+    
+    STR_SHORTCUT_OPEN_CLEAR_SCENERY=6131,
```
Comment (marijnvdwerf) [inline src/openrct2/localisation/string_ids.h]: Missing spaces around `=` symbol (`SCENERY = 6131`)

File: src/openrct2/interface/window.h (hunk 1 above)
Comment (Kuriibo) [inline src/openrct2/interface/window.h]: `validate_global_widx(WC_TOP_TOOLBAR, WIDX_CLEAR_SCENERY)` has been added to `top_toolbar.c`

**Pull request merged**